crates/ruff_workspace/src/configuration.rs

```diff
@@ -968,6 +973,20 @@ impl LintConfiguration {
             }
         }
 
+        if !removed_ignored_rules.is_empty() {
+            let mut rules = String::new();
+            for selection in removed_ignored_rules.iter().sorted() {
+                let (prefix, code) = selection.prefix_and_code();
+                rules.push_str("\n    - ");
+                rules.push_str(prefix);
+                rules.push_str(code);
+            }
+            rules.push('\n');
+            warn_user_once_by_message!(
+                "The following rules have been removed and ignoring them is useless:{rules}"
```

Comment:
Somehow "useless" sounds harsh, how about:

```diff
- "The following rules have been removed and ignoring them is useless:{rules}"
+ "The following rules have been removed and ignoring them has no effect:{rules}"
```

Reply:
I used `useless` because that's what we also use in many rules. Another terminology that we commonly use in rules is unnecessary, would you like that better?

Reply:
I really like Dylan's wording of "has no effect". Even "unnecessary" sounds slightly judgemental to me haha

Reply:
Fine :joy: